crates/oxc_linter/src/rules/unicorn/no_empty_file.rs

```diff
@@ -58,7 +58,7 @@ impl Rule for NoEmptyFile {
             return;
         }
 
-        ctx.diagnostic(no_empty_file_diagnostic(Span::new(0, 0)));
+        ctx.diagnostic(no_empty_file_diagnostic(program.span));
```

Comment:
This will spit out a huge diagnostic if the file has lots of commented code. Pointing to the first line should be fine here?

Reply:
I've limited it to the first 100 characters to avoid searching for the new line.
Disable comments without any leading or trailing spaces and with only a single rule(`no-empty-file`) is 34 characters, So a hundred should be plenty for multi-rule comments or weirdly formatted ones.

Let me know if this doesn't work for you or if you insist on limiting it to exactly the first line.